<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessKnightPathFinder</class>
 <widget class="QMainWindow" name="ChessKnightPathFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessKnightPathFinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="chessboard"/>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
font: 81 16pt &quot;Arial&quot; bold;</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStart">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(152, 106, 68);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Введите начальную позицию</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEnd">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(152, 106, 68);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Введите конечную позицию</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
